addons/mail/models/mail_followers.py

```diff
@@ -508,3 +509,20 @@ def _add_followers(self, res_model, res_ids, partner_ids, subtypes,
                         update[fol_id] = {'subtype_ids': update_cmd}
 
         return new, update
+
+    def _follower_fetch(self, domain, after=None, limit=100):
+        if after:
+            domain = expression.AND([domain, [('id', '>', after)]])
+        follower_ids = self.search(domain, limit=limit, order='id ASC')
+        return follower_ids
+
+    def _follower_format(self):
```

Comment:
Follow keyword is not necessary, you are on follower model. Maybe ``_format_for_chatter`` or something like that ?